data-processing/common/commands/fetch_s2_data.py

```diff
@@ -81,3 +83,43 @@ def save(self, item: ArxivId, result: S2Metadata) -> None:
         s2_id_path = os.path.join(s2_metadata_dir, "s2_id")
         with open(s2_id_path, "w") as s2_id_file:
             s2_id_file.write(result.s2_id)
+
+    def _strip_arxiv_version(self, item: ArxivId) -> ArxivId:
+        """Remove any version identifier from the passed ArXiv ID.
+        Looks for the last 'v' (case-insensitive) and strips beyond it.
```

Comment:
Why search strip after the last v and not the first? Is the reasoning that there might be a 'v' in some old-style arXiv IDs?

Reply:
Actually you just made me realize something I missed, will update the PR for identifiers like "math.VX/12133" where they are not versioned but the old ArXiv ID has a "v" in the sub-field piece (VX is made-up here).

Specifically to your question, I just think this case isn't going to happen in practice and so I just picked one way and added the example to illustrate.